src/elm/Claim.elm

```diff
@@ -270,20 +273,23 @@ updateClaimModalStatus msg model =
         OpenPhotoModal claimId ->
             { model | claimModalStatus = PhotoModal claimId }
 
+        OpenClaimModal claimId ->
+            { model | claimModalStatus = ClaimModal claimId }
+
         RouteOpened _ ->
             model
 
         GotProfileSummaryMsg _ ->
             model
-
-
 {-| Claim card with a short claim overview. Used on Dashboard and Analysis pages.
 -}
-viewClaimCard : LoggedIn.Model -> Profile.Summary.Model -> Model -> Html Msg
-viewClaimCard { shared, accountName } profileSummary claim =
+
+
+viewClaimCard : LoggedIn.Model -> Profile.Summary.Model -> Model -> Bool -> Html Msg
+viewClaimCard loggedIn profileSummary claim showClaimModal =
     let
-        { t } =
-            shared.translators
+        t =
+            loggedIn.shared.translators.t
```

Comment:
Is there a reason for this? Most pages use the `{ t }` format so it's easier to get more things (e.g. `tr`) when needed. I know it's not a big deal, but it's usually good to follow conventions used throughout the code base